operator/gefyra/resources/deployments.py

```diff
@@ -25,6 +26,13 @@ def create_stowaway_deployment() -> k8s.client.V1Deployment:
             requests={"cpu": "0.1", "memory": "100Mi"},
             limits={"cpu": "0.75", "memory": "500Mi"},
         ),
+        readiness_probe=V1Probe(
+            _exec=V1ExecAction(
+                command=["cat", "/config/peer1/peer1.conf"],
+            ),
+            period_seconds=1,
+            initial_delay_seconds=5,
```

Comment:
I'd say we can set this to 1, why wait 5 seconds (on very fast systems this may become unnecessarily long)?

Reply:
Lowered `initial_delay_seconds`.